### ISSUE TITLE
Full-screen Multimonitor Bug

### ISSUE THREAD

**Issue by fifothekid** [FreeRDP/FreeRDP#4786]
I've been using the latest master version for months now to access my home computer with 3 monitors using the following command from Ubuntu 16.04:
xfreerdp +auto-reconnect /frame-ack:0 /bpp:16 -decorations /dynamic-resolution /f +fonts /gdi:hdi +home-drive -themes -wallpaper +clipboard /u:*** /multimon:3 /p:*** /rfx /rfx-mode:video +drives /disp /v:201.**.**.**:3389
It has been working perfectly but when I downloaded the latest updates to the master branch today and compiled I started having a bug where starting the remote connection with the command above works perfectly, but when I exit full-screen mode FreeRDP would switch to single monitor mode and it would never work on multiple monitors again until I restart the whole application.
This definitely didn't happen up to two weeks ago.

**Comment by hardening** [comment]
do you get the same if you don't use `/dynamic-resolution` ?

**Comment by fifothekid** [comment]
removing /dynamic-resolution solved it

**Comment by hardening** [comment]
@akallabeth could it be related to you last changes in the dynamic channel ?

**Comment by akallabeth** [comment]
@hardening No, just checked. With the old version `/dynamic-resolution` did not work with fullscreen as the window was non resizeable.
It now is working but the restore fullscreen logic was never adapted to restore multimonitor fullscreen.

**Comment by hardening** [comment]
@fifothekid does it work with @akallabeth 's fix applied ?
